vendor/engines/projects/app/views/projects/projects/index.html.haml

```diff
@@ -0,0 +1,17 @@
+= content_for :h1 do
+  = t(".head")
+
+- if current_ability.can?(:create, Projects::Project)
+  %p= link_to t(".add"), new_facility_project_path, class: "btn-add"
+
+- if @projects.any?
+  %table.table.table-striped.table-hover
+    %thead
+      %tr
+        %th= Projects::Project.human_attribute_name(:name).titlecase
```

Comment:
Maybe? I think the chance of this translation changing to something that's more than one word is low, but if it did become 2 or more words, would we want it `titlecase`d?

There are other places where we have header labels with more than one word, and that's what we're doing. One example:
https://github.com/tablexi/nucore-open/blob/master/app/views/facility_journals/index.html.haml#L54-L55